public/modules/custom/hel_tpm_editorial/js/hel-tpm-datetimepicker.js

```diff
@@ -9,8 +9,8 @@
         if (placeholder === null) {
           return;
         }
-        if (placeholder === 'hh:mm:ss') {
-          $(this).attr('placeholder',  'hh.mm');
+        if (placeholder === 'hh:mm:ss' || placeholder === 'hh:mm') {
+          $(this).attr('placeholder',  'hh:mm');
```

Comment:
Could this work with the correct Finnish format (hh.mm) instead?

Reply:
Committed a suggested fix. The form, however, doesn't work – should it, or is it broken ATM?